CMakeLists.txt

```diff
@@ -1,119 +1,207 @@
 #
-#    This file is part of acados.
+#	 This file is part of acados.
 #
-#    acados is free software; you can redistribute it and/or
-#    modify it under the terms of the GNU Lesser General Public
-#    License as published by the Free Software Foundation; either
-#    version 3 of the License, or (at your option) any later version.
+#	 acados is free software; you can redistribute it and/or
+#	 modify it under the terms of the GNU Lesser General Public
+#	 License as published by the Free Software Foundation; either
+#	 version 3 of the License, or (at your option) any later version.
 #
-#    acados is distributed in the hope that it will be useful,
-#    but WITHOUT ANY WARRANTY; without even the implied warranty of
-#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
-#    Lesser General Public License for more details.
+#	 acados is distributed in the hope that it will be useful,
+#	 but WITHOUT ANY WARRANTY; without even the implied warranty of
+#	 MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+#	 Lesser General Public License for more details.
 #
-#    You should have received a copy of the GNU Lesser General Public
-#    License along with acados; if not, write to the Free Software Foundation,
-#    Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+#	 You should have received a copy of the GNU Lesser General Public
+#	 License along with acados; if not, write to the Free Software Foundation,
+#	 Inc., 51 Franklin Street, Fifth Floor, Boston, MA	02110-1301	USA
 #
 #
 
 cmake_minimum_required(VERSION 3.1)
 
 if(CMAKE_MAKE_PROGRAM)
-    file(TO_CMAKE_PATH ${CMAKE_MAKE_PROGRAM} CMAKE_MAKE_PROGRAM)
-    find_program(CMAKE_MAKE_PROGRAM ${CMAKE_MAKE_PROGRAM})
+	file(TO_CMAKE_PATH ${CMAKE_MAKE_PROGRAM} CMAKE_MAKE_PROGRAM)
+	find_program(CMAKE_MAKE_PROGRAM ${CMAKE_MAKE_PROGRAM})
 endif()
 
 project(acados LANGUAGES C CXX)
 
 list(APPEND CMAKE_MODULE_PATH "${PROJECT_SOURCE_DIR}/cmake")
+
 set(CMAKE_POSITION_INDEPENDENT_CODE TRUE)
 
+### Defaults
+
+set(ACADOS_INSTALL_DIR "/usr/local" CACHE STRING  "Installation path to /usr/local")
+set(BLASFEO_TARGET "GENERIC" CACHE STRING "BLASFEO Target architecture")
+set(HPIPM_TARGET "GENERIC" CACHE STRING "HPIPM Target architecture")
+set(LA "HIGH_PERFORMANCE" CACHE STRING "Linear algebra optimization level")
+
+# Additional targets
+option(ACADOS_UNIT_TESTS "Compile Unit tests" OFF)
+option(ACADOS_EXAMPLES "Compile Examples" OFF)
+option(ACADOS_LINT "Compile Lint" OFF)
+# Extarnal libs
+option(ACADOS_WITH_QPOASES  "qpOASES solver" ON)
```

Comment:
`OFF` should be the default, as `qpOASES` is not a dependency

Reply:
ok, see related [comment](https://github.com/acados/acados/pull/393#discussion_r289375876)

Reply:
fixed